bot/exts/evergreen/profile_pic_modification/pfp_modify.py

```diff
@@ -0,0 +1,236 @@
+            egg = None
+            if colours:
+                send_message = ctx.send
+                ctx.send = send  # Assigns ctx.send to a fake send
+                egg = await ctx.invoke(self.bot.get_command("eggdecorate"), *colours)
+                if isinstance(egg, str):  # When an error message occurs in eggdecorate.
+                    await send_message(egg)
+                    return
+                ctx.send = send_message  # Reassigns ctx.send
+
+            image_bytes = await ctx.author.avatar_url_as(size=256).read()
+            file = await in_thread(
+                PfpEffects.apply_effect,
+                image_bytes,
+                PfpEffects.easterify_effect,
+                egg
+            )
+
+            embed = discord.Embed(
+                name="Your Lovely Easterified Avatar",
+                description="Here is your lovely avatar, all bright and colourful\nwith Easter pastel colours. Enjoy :D"
+            )
+            embed.set_image(url="attachment://modified_avatar.png")
+            embed.set_footer(text=f"Made by {ctx.author.display_name}", icon_url=ctx.author.avatar_url)
+
+        await ctx.send(file=file, embed=embed)
+
+    async def send_pride_image(
+        self,
+        ctx: commands.Context,
+        image_bytes: bytes,
+        pixels: int,
+        flag: str,
+        option: str
+    ) -> None:
+        """Gets and sends the image in an embed. Used by the pride commands."""
+        async with ctx.typing():
+            file = await in_thread(
+                PfpEffects.apply_effect,
+                image_bytes,
+                PfpEffects.pridify_effect,
+                pixels,
+                flag
+            )
+
+            embed = discord.Embed(
+                name="Your Lovely Pride Avatar",
+                description=f"Here is your lovely avatar, surrounded by\n a beautiful {option} flag. Enjoy :D"
+            )
+            embed.set_image(url="attachment://modified_avatar.png")
+            embed.set_footer(text=f"Made by {ctx.author.display_name}", icon_url=ctx.author.avatar_url)
+            await ctx.send(file=file, embed=embed)
+
+    @pfp_modify.group(
+        aliases=["avatarpride", "pridepfp", "prideprofile"],
+        root_aliases=("prideavatar", "avatarpride", "pridepfp", "prideprofile"),
+        invoke_without_command=True
+    )
+    async def prideavatar(self, ctx: commands.Context, option: str = "lgbt", pixels: int = 64) -> None:
+        """
+        This surrounds an avatar with a border of a specified LGBT flag.
+
+        This defaults to the LGBT rainbow flag if none is given.
+        The amount of pixels can be given which determines the thickness of the flag border.
+        This has a maximum of 512px and defaults to a 64px border.
+        The full image is 1024x1024.
+        """
+        option = option.lower()
+        pixels = max(0, min(512, pixels))
+        flag = self.GENDER_OPTIONS.get(option)
+        if flag is None:
```

Comment:
Do you not think have it written like `if flag is None` is easier to read? As it specifically stating the case in which the if block is running, rather than the inverse.

Reply:
Well that’s my preference and I find it more readable.